frontend/src/hooks/useWebMedia.ts

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react'
+
+export const audioOptions = { audio: true, video: false }
+export const videoOptions = { audio: true, video: true }
+export const photoOptions = { audio: true, video: false }
+
+export const useWebMediaRecorder = (options: any) => {
+  const [audioURL, setAudioURL] = useState({})
+  const [isRecording, setIsRecording] = useState(false)
+  const [recorder, setRecorder]: any = useState(null)
```

Comment:
We could also add a type here like so:

```suggestion
  const [recorder, setRecorder] = useState<MediaRecorder | undefined>(undefined)
```

However, we would have to add conditional `?´ to the `recording` variable everywhere. Not sure if worth it but I like types. But also fine if it stays the way it is.